xml-parser/src/test/java/org/uniprot/core/xml/AbstractConverterTest.java

```diff
@@ -0,0 +1,39 @@
+package org.uniprot.core.xml;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import javax.xml.datatype.DatatypeFactory;
+import javax.xml.datatype.XMLGregorianCalendar;
+
+import org.bbottema.loremipsumobjects.ClassBindings;
+import org.bbottema.loremipsumobjects.LoremIpsumObjectCreator;
+import org.bbottema.loremipsumobjects.typefactories.MethodBasedFactory;
+import org.uniprot.core.xml.jaxb.unirule.ObjectFactory;
+
+public abstract class AbstractConverterTest {
```

Comment:
The intent of this abstract class is to make sure that everybody in UniRule converter test extends it and it cannot be thought as independent class. In the end i have added Test to avoid people thinking that it is a main production code.